### PR TITLE
Proper header according to the specific role

### PR DESCRIPTION
**Pull request by Winchestery**
Proper header with routing according to the specific defined role and logging out

### CODE REVIEW

File: src/features/header/header.js [hunk 1]
```diff
@@ -21,24 +30,68 @@ const sidebarToggler = (
...
   
-  const [tabs, setTabs] = useState([
-    {id: 0, title: 'Progress', link: 'progress', active: false},
-    {id: 1, title: 'Schedule', link: 'schedule', active: false},
-    {id: 2, title: 'Support', link: 'support', active: false},
-  ]);
-
+  const rolesObject = {
+    4: [
+      {id: 0, title: 'Students', link: 'students', active: true},
```
Comment (PavelKundenko) [inline src/features/header/header.js:38]: Please, use constants from paths.js as links
```
{id: 0, title: 'Students', link: paths.STUDENTS, active: true},
```

File: src/features/header/header.js [hunk 2]
```diff
@@ -21,24 +30,68 @@ const sidebarToggler = (
...
   const [sidebar, setSidebar] = useState({
     active: false,
   });
-
+  const definedRole = Object.keys(rolesObject).find(el =>  Number(el) === currentUser.role)
+  useEffect(() => {
+    const headerArray = rolesObject[definedRole]
+    setTabs(headerArray)
+  
+  }, [definedRole])
```
Comment (PavelKundenko) [inline src/features/header/header.js:87]: Seems to me that we can simplify: 
```
  useEffect(() => {
    const headerArray = rolesObject[currentUser.role]
    setTabs(headerArray)
  }, [currentUser])
```

File: src/components/protected-route/protected-route.js [hunk 3]
```diff
@@ -18,6 +18,7 @@ export const ProtectedRoute = ({ roles, ...otherProps }) => {
   if (!roles.includes(currentUser.role)) {
     return <Redirect to={paths.NOT_FOUND} />;
   }
+  // console.log(roles)
```
Comment (PavelKundenko) [inline src/components/protected-route/protected-route.js:21]: Unnecessary comment

**Review by Icemandov — APPROVED** [review]

File: src/features/header/header.js (hunk 2 above)
Comment (Winchestery) [reply src/features/header/header.js:87]: done

File: src/components/protected-route/protected-route.js (hunk 3 above)
Comment (Winchestery) [reply src/components/protected-route/protected-route.js:21]: done

File: src/features/header/header.js (hunk 1 above)
Comment (Winchestery) [reply src/features/header/header.js:38]: done

**Review by so2niko — APPROVED** [review]

**Review by PavelKundenko — APPROVED** [review]

**Review by sarhan-azizov — APPROVED** [review]

**Pull request merged**